src/FormsySelect.jsx

```diff
@@ -37,18 +37,20 @@ const FormsySelect = React.createClass({
   setMuiComponentAndMaybeFocus: setMuiComponentAndMaybeFocus,
 
   render() {
-    let { value } = this.props;
+    const { 
+      requiredError,
+      validations, // eslint-disable-line no-unused-vars
+      validationError, // eslint-disable-line no-unused-vars
+      validationErrors, // eslint-disable-line no-unused-vars
+      value: valueProp,
+      ...rest 
+    } = this.props;
 
-    const { validations, // eslint-disable-line no-unused-vars
-        validationError, // eslint-disable-line no-unused-vars
-        validationErrors, // eslint-disable-line no-unused-vars
-      ...rest } = this.props;
-
-    value = this.state.hasChanged ? this.getValue() : value;
-    const { requiredError } = this.props;
     const { isRequired, isPristine, isValid, isFormSubmitted } = this;
     const isRequiredError = isRequired() && !isPristine() && !isValid() && isFormSubmitted() && requiredError;
     const errorText = this.getErrorMessage() || isRequiredError;
+    value = this.state.hasChanged ? this.getValue() : valueProp;
```

Comment:
I am getting `ReferenceError: value is not defined`

You are missing the variable declaration